test/smoke/run-smoke-suite.sh

```diff
@@ -184,13 +183,13 @@ spec:
     - {name: WALLARM_API_HOST, value: "${WALLARM_API_HOST}"}
     - {name: WALLARM_API_PRESET, value: "${WALLARM_API_PRESET:-eu1}"}
     - {name: API_CA_VERIFY, value: "${WALLARM_API_CA_VERIFY}"}
-    - {name: CLIENT_ID, value: "${CLIENT_ID}"}
+    - {name: CLIENT_ID, value: "${CLIENT_ID:-5}"}
```

Comment:
let's put all defaults from here to line 25 including API_PRESET

Reply:
fixed